lib/slack/web/api/templates/method.erb

```diff
@@ -51,6 +51,9 @@ module Slack
 <% if data['deprecated'] %>
             logger.warn('<%= group %>.<%= name %>: <%= data['deprecation']['deprecation_warning']%> Alternative methods: <%= data['deprecation']['alternative_methods'].join(', ')%>.')
 <% end %>
+<% if %w[chat views dialog].include?(group) %>
```

Comment:
I don't love that this array of strings will need to be updated in coordination with `encode_options` when new groups are added, but I also wasn't keen to add this line to all endpoints.

Reply:
Can we promote `encoding: json` or a similar flag into slack-api-ref to avoid this?